indico/web/templates/header.html

```diff
@@ -9,8 +9,12 @@
     <div class="flexrow f-j-space-between f-a-start">
         {% block logo %}
             {%- set logo_url = indico_config.LOGO_URL or (indico_config.IMAGES_BASE_URL + '/logo_indico_bw.svg') -%}
-            <a style="min-height: 60px;" href="{{ url_for_index() }}">
-                <img class="header-logo" src="{{ logo_url }}">
+            <a id="header-logo"
+               href="{{ url_for_index() }}"
+               aria-label="{{ _('Indico main page') }}"
+               aria-current="{{ 'true' if url_for_index() == request.path else 'false' }}"
```

Comment:
Not a fan of this way to check for the main page, but I can't find a better alternative ;x

For the boolean conversion you could use `tojson`, but no super strong opinion here since it's not really JSON.

```suggestion
               aria-current="{{ (url_for_index() == request.path) | tojson }}"
```

Reply:
> For the boolean conversion you could use tojson, but no super strong opinion here since it's not really JSON.

Yeah, I thought it would be confusing. I could whip up a `truefalse` filter, though. It might look nicer and less verbose. WDYT?

Reply:
Actually, there's no need. The attribute value is case-insensitive anyway. 😂